### ISSUE TITLE
Are the models in drive already trained on Adversarial examples generated?

### ISSUE THREAD

**Issue by SachJbp** [jind11/TextFooler#27]
I wanted to ask if the models in Google drive are already trained on Adversarial examples which you have generated?

Thanks

**Comment by jind11** [comment]
no, they are not. They are merely trained on the original clean data.

**Comment by SachJbp** [comment]
Thanks for the clarification.
